app/views/job_invocations/_tab_overview.html.erb

```diff
@@ -27,7 +27,12 @@
     <h5>
       <b><%= template_invocation.template.name %></b> <%= 'through' %> <%= _(RemoteExecutionProvider.provider_for(template_invocation.template.provider_type)) %>
     </h5>
-    <pre><%= InputTemplateRenderer.new(template_invocation.template, nil, template_invocation).preview %></pre>
+    <% target = template_invocation.targeting.hosts.first %>
```

Comment:
I incline to solve this through inputs somehow. This bug was more about completely missing host, so even if host had operating system, @host.operatingsystem resulted on calling operatingsystem on nil... Anyway I changed it to prefer host with OS for now and fallback to any host from targeting.